projects/plugins/jetpack/package.json

```diff
@@ -43,6 +43,7 @@
 		"test-e2e": "NODE_CONFIG_DIR='./tests/e2e/config' NODE_CONFIG_ENV=test JEST_PUPPETEER_CONFIG=tests/e2e/jest-puppeteer.config.js jest --config tests/e2e/jest.config.js --runInBand --verbose",
 		"test-encrypt-config": "openssl enc -md sha1 -aes-256-cbc -pass env:CONFIG_KEY -in ./tests/e2e/config/local-test.js -out ./tests/e2e/config/encrypted.enc",
 		"test-extensions": "jest extensions",
+		"test-extensions-single": "jest",
```

Comment:
Is there a benefit from running a single test with this command VS `yarn jest path/to/file`?

Reply:
🤦 Doh! nope, no point at all in that, not sure why `yarn jest path/to/file` didn't stand out as the obvious solution to that on the day, must have been before coffee - have removed.